client/state/notices/reducer.js

```diff
@@ -31,6 +32,16 @@ export const items = createReducer( {}, {
 
 		return omit( state, noticeId );
 	},
+	[ NOTICE_CLEAR ]: ( state ) => {
+		return reduce( state, ( memo, notice, noticeId ) => {
```

Comment:
For a `NOTICE_CLEAR` action, I would have expected a simple `return {};` . It seems confusing that persistent notices stick around. Should we indicate this in the action creator name? Or parameterize it if there's use cases where those persistent notices should stick around?